app/models/post.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
+  validates :context, presence: true, length: { maximum: 250, minimum: 5 }
+  validates :user_id, presence: true
```

Comment:
- [ ] `belongs_to` association already adds presence validation to `user_id`. We don't need it.